app/views/staff/shared/transactions/_table.html.haml

```diff
@@ -2,31 +2,19 @@
   %table.govuk-table.transactions
     %thead.govuk-table__head
       %tr.govuk-table__row
-        %th.govuk-table__header
```

Comment:
Ah, we've ended up working in parallel on the same thing and duplicating some work.

This card https://trello.com/c/O13SejbU/1299-the-actuals-transactions-summary-is-consistent-with-the-forecasts-summary is specifically about removing these fields from the view and replacing the date with quarter, so my PR being merged will create even more conflicts for you. :/

Let me know if you have any trouble with the conflicts, I'll be available to help.

Reply:
I see! I'll have a look and give you a shout if I need help. Thanks Cristina!! :-)

Reply:
Okay, I've rebased and I think everything looks the way it should, but please double check and let me know if that's not the case.